src/Engine/labyrinthFactory.h

```diff
@@ -0,0 +1,25 @@
+//
+// Created by nikit on 15/04/2018.
+//
+
+#ifndef ALGODATII_LABYRINTHFACTORY_H
+#define ALGODATII_LABYRINTHFACTORY_H
+
+typedef struct labyrinth {
```

Comment:
typedef ist unnötig...

Reply:
Es ist hilfreich Aussagen zu begründen.

Reply:
Struct und class sind, wie in Whatsapp bereits geschrieben, (fast) das Gleiche in C++.

Zu typedef:
Typedef ist, wie in C, dazu da Typ-Aliase zu schaffen. In C ist die Syntax zum Erstellen einer Instanz <code>struct Typname Instanzname</code>. In C++ einfach nur <code>Typname Instanzname.</code>
Der Typedef <code>typedef struct { ... } Typname;</code> in C ist also ein Hack, um sich Schreibarbeit zu sparen. In C++ wird das aber nicht mehr benötigt.